client/blocks/app-banner/test/app-banner.jsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * @format
+ * @jest-environment jsdom
+ */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
```

Comment:
Let's avoid introducing new usage of `chai`.

- Remove this import line.
- Jest will provide the global `expect` –no additional action required
- `expect( … ).equal( 'some-string' )` becomes `expect( … ).toBe( 'some-string' )`

Reply:
Thanks – I've pushed a fix for this.